File: rb/test/exists_test.rb
# RonSwansonQuotes SDK exists test

require "minitest/autorun"
require_relative "../RonSwansonQuotes_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RonSwansonQuotesSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- RonSwansonQuotes SDK exists test

local sdk = require("ron-swanson-quotes_sdk")

describe("RonSwansonQuotesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
